src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,16 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
```

Comment:
Consider adding a private constructor to prevent instantiation of this utility class. This is a common practice for utility classes to ensure they are not instantiated.